### DISCUSSION TITLE
Welcome to UHSDR Discussions!

### DISCUSSION THREAD

**Discussion by df8oe** [df8oe/UHSDR#1883] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WB8HSL** [comment]
Dear All,

Is there a log that shows in plain language what the newest update(s) add or fix?

Thank you,

Scott
